deploy-minikube.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -x
+minikube delete
```

Comment:
locally this will remove the cluster, so the VMs will be removed.
Then installed again. This will take a long time, I would prefer keeping the VMs installed and only removing deployments.

Reply:
This is for the whole workflow. we can run the scripts individually. For instance just do make clean and run the script for partner, test and operator (not the build operator since it is already saved as a tar file)

Reply:
can we have something like 
make deploy -> deploy if minikube is already installed
make clean-deployment -> clean deployment only 
?
It would be better to have some something easy to use when we have minikube installed, rather than calling every script separately

Reply:
fixed